app/services/extractors/open_neuro.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Extractors
```

Comment:
Instead of a new extractor, I'd just recommend making the Datacite extractor and etl job accept the client_id parameter. Then you can have multiple instances of the job scheduled (similar to redivis).